.travis.yml

```diff
@@ -5,3 +5,7 @@ install:
 - pip install -r requirements.txt
 script:
 - pytest
+- pre-commit run --all-files
```

Comment:
I wonder if this will fail because I didn't commit/fix the changes on `alternate_scrape_function.py` ...we shall see